File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Geometry.java

package restaurant.sa.com.sarestaurant.appview.restaurant.model;

import com.google.gson.annotations.Expose;

public class Geometry {

    @Expose
    private Location location;
    @Expose
    private Viewport viewport;

    public Location getLocation() {
        return location;
    }

    public void setLocation(Location location) {
        this.location = location;
    }

    public Viewport getViewport() {
        return viewport;
    }

    public void setViewport(Viewport viewport) {
        this.viewport = viewport;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Geometry.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Embedded
import android.arch.persistence.room.Entity

@Entity(tableName = "Geometry")
data class Geometry(
        @Embedded var location: Location?,
        @Embedded var viewport: Viewport?)